arisa.json

```diff
@@ -48,7 +48,7 @@
           "Won't Fix",
           "Works As Intended"
         ],
-        "extensionBlacklist": [
+        "excludedExtensions": [
```

Comment:
"Excluded" is a little bit irritating because the purpose of the module is to remove attachments and the extensions here are not actually excluded but included for removal.
Maybe "disallowedExtensions" might be better?